lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -483,13 +481,13 @@ def test_scatter3d_modification(fig_ref, fig_test):
                    depthshade=False, s=75, linewidths=3)
 
 
-@pytest.mark.parametrize('depthshade', [True, False])
```

Comment:
The new depthshade behavior applies depth shading for each Patch3DCollection. Since we are splitting this reference image into 4x scatter commands, it will have different depth shading than the single 1x scatter command for the test image. 

![image](https://github.com/user-attachments/assets/48a8dc15-74f5-4f9f-a960-5fe9630edc89)

I think this per-collection depth shading makes the most sense. The alternative would be to apply depth shading based on a fixed distance from the camera, which would give transparency to equal-distance points near the far clipping plane of the camera. Perhaps that is more intuitive for some people? But IMO the depth shading is a visual cue to help distinguish between points, so the absolute shading should be as opaque as possible, and the transparency only applied to show relative shading.

The middle ground would be to track all the patches in the scene and apply depth shading based on all the Patch3DCollections collectively, but I think that's probably hard to handle internally.

The original PR which added this test (https://github.com/matplotlib/matplotlib/pull/18293/) was only concerned with the depthshading=False case, so I don't think we lose much by removing this one.

Reply:
Of note, the behavior before this current PR did not seem to be showing depthshading at all during the depthshading=True test case. I'm not sure why.

![main branch Screenshot 2024-12-21 145212](https://github.com/user-attachments/assets/5f102c1a-78e7-41dc-b36b-5a7836bcf9cb)